build.rs

```diff
@@ -1,10 +1,17 @@
 extern crate prost_build;
 
 fn main() {
-    std::env::set_var(
-        "PROTOC",
-        protoc_bin_vendored::protoc_bin_path().expect("protoc binary not found"),
-    );
+    if std::env::var_os("PROTOC")
```

Comment:
Can we match on `protoc_bin_vendored::protoc_bin_path()` first, and only go for existing installs as fallback if the bundled one does not exist? :)

Reply:
ok, so I can check only if protoc_bin_path exists and in that case i assign the protoc path, otherwise I let it panic in case protoc is not installed